yarn.lock

```diff
@@ -5008,10 +5008,10 @@
   dependencies:
     "@types/d3-path" "^1"
 
-"@types/d3-time-format@^2.1.0":
-  version "2.1.1"
-  resolved "https://registry.yarnpkg.com/@types/d3-time-format/-/d3-time-format-2.1.1.tgz#dd2c79ec4575f1355484ab6b10407824668eba42"
-  integrity sha512-tJSyXta8ZyJ52wDDHA96JEsvkbL6jl7wowGmuf45+fAkj5Y+SQOnz0N7/H68OWmPshPsAaWMQh+GAws44IzH3g==
+"@types/d3-time-format@3.0.0":
```

Comment:
Have you run dedupe in this file

Reply:
yes, 
<img width="310" alt="image" src="https://github.com/microsoft/fluentui/assets/120183316/bf7b8707-1bf1-4804-8e16-d55d4fecda07">